### PR TITLE
Fix logLevel check in chart

### PR DESCRIPTION
**Pull request by AKamyshnikova**
| Q               | A
| --------------- | ---
| Bug fix?        | [x]
| New feature?    | []
| API breaks?     | []
| Deprecations?   | []
| Related tickets | fixes #X, partially #Y, mentioned in #Z
| License         | Apache 2.0

### CODE REVIEW

**Comment by AKamyshnikova** [comment]
Checked with e2e tests https://github.com/AKamyshnikova/casskop/actions/runs/9226953689

File: controllers/cassandracluster/generator.go [hunk 1]
```diff
@@ -925,6 +925,12 @@ func createCassandraContainer(cc *api.CassandraCluster, status *api.CassandraClu
 	} else {
 		command = []string{"cassandra", "-f"}
 	}
+	cassandraEnv := commonBootstrapCassandraEnvVar(cc)
+	// This option required for nodetool correct execution
+	cassandraEnv = append(cassandraEnv, v1.EnvVar{
+		Name:  "JAVA_TOOL_OPTIONS",
+		Value: "-Dcom.sun.jndi.rmiURLParsing=legacy",
+	})
```
Comment (cscetbon) [inline controllers/cassandracluster/generator.go:933]: Let's make this its own PR

**Review by cscetbon — APPROVED** [review]
